test/acceptance/authz/rbac_viewer_test.go

```diff
@@ -43,9 +43,9 @@ func TestViewerEndpoints(t *testing.T) {
 		{endpoint: "authz/users/id/roles", methods: []string{"GET"}, success: []bool{true}, arrayReq: false},
 		{endpoint: "authz/users/id/assign", methods: []string{"POST"}, success: []bool{false}, arrayReq: false},
 		{endpoint: "authz/users/id/revoke", methods: []string{"POST"}, success: []bool{false}, arrayReq: false},
-		{endpoint: "backups/backend", methods: []string{"GET", "POST"}, success: []bool{true, false}, arrayReq: false},
-		{endpoint: "backups/backend/id", methods: []string{"GET", "DELETE"}, success: []bool{true, false}, arrayReq: false},
-		{endpoint: "backups/backend/id/restore", methods: []string{"GET", "POST"}, success: []bool{true, false}, arrayReq: false},
+		// {endpoint: "backups/backend", methods: []string{"GET", "POST"}, success: []bool{true, false}, arrayReq: false},
```

Comment:
**question (if-minor):** is this meant to be commented out ?

Reply:
No! Thanks for the catch 😁